tests/python/gaia-ui-tests/gaiatest/mixins/gaia.py

```diff
@@ -21,6 +21,21 @@ class GaiaArguments(object):
          {'default': "undefined",
           'help': 'locale for the device, This value overrides the value from testvars.json file',
           }],
+        [['--capture'],
+         {'dest': 'capture',
+          'default': "whenfail",
```

Comment:
aha, I thought you wanted it whenfail by default, from the title of the bug.  will do.